bundles/best_practices.rst

```diff
@@ -107,27 +118,27 @@ and others are just conventions followed by most developers):
 ===================================================  ========================================
 Type                                                 Directory
 ===================================================  ========================================
-Commands                                             ``Command/``
-Controllers                                          ``Controller/``
-Service Container Extensions                         ``DependencyInjection/``
-Doctrine ORM entities (when not using annotations)   ``Entity/``
-Doctrine ODM documents (when not using annotations)  ``Document/``
-Event Listeners                                      ``EventListener/``
-Configuration (routes, services, etc.)               ``Resources/config/``
-Web Assets (CSS, JS, images)                         ``Resources/public/``
-Translation files                                    ``Resources/translations/``
-Validation (when not using annotations)              ``Resources/config/validation/``
-Serialization (when not using annotations)           ``Resources/config/serialization/``
-Templates                                            ``Resources/views/``
-Unit and Functional Tests                            ``Tests/``
+Commands                                             ``src/Command/``
+Controllers                                          ``src/Controller/``
+Service Container Extensions                         ``src/DependencyInjection/``
+Doctrine ORM entities (when not using annotations)   ``src/Entity/``
+Doctrine ODM documents (when not using annotations)  ``src/Document/``
+Event Listeners                                      ``src/EventListener/``
+Configuration (routes, services, etc.)               ``config/``
+Web Assets (CSS, JS, images)                         ``public/``
+Translation files                                    ``translations/``
+Validation (when not using annotations)              ``config/validation/``
+Serialization (when not using annotations)           ``config/serialization/``
+Templates                                            ``templates/``
+Unit and Functional Tests                            ``tests/``
 ===================================================  ========================================
 
 Classes
 -------
 
 The bundle directory structure is used as the namespace hierarchy. For
 instance, a ``ContentController`` controller which is stored in
-``Acme/BlogBundle/Controller/ContentController.php`` would have the fully
+``Acme/BlogBundle/src/Controller/ContentController.php`` would have the fully
```

Comment:
there is no `Acme/BlogBundle/` folder there before `src`. that's  left-over from old times